obspy/io/nordic/tests/test_nordic.py

```diff
@@ -546,6 +587,18 @@ def test_distance_conversion(self):
             int([p for p in pick_strings if p.split()[0] == 'WZ11' and
                  p.split()[1] == 'HZ'][0].split()[-1]), 30)
 
+    def test_convert_xyz_error(self):
+        """
+        Test that we are able to map between xyz errors and ellipsoid
+        """
+        errors = {'x_err': 1., 'y_err': 1.5, 'z_err': 2.0,
+                  'xy_cov': 0.0, 'xz_cov': 0.0, 'yz_cov': 0.0}
+        confidence_ellipsoid = xyz_to_confidence_ellipsoid(errors)
+        print(confidence_ellipsoid)
+        errors_back = confidence_ellipsoid_to_xyz(confidence_ellipsoid)
+        self.assertEqual(errors, errors_back)
+        # TODO: Test mapping for known result.
```

Comment:
This should really be done as its otherwise hard to really trust the calculations - so the test just shows that one function is the inverse of the other which is necessary but not sufficient IMHO. For simple cases it should also be fairly easy to do.

Reply:
Agree - sorry, this still isn't ready - the confidence ellipsoid stuff still doesn't get the right result for the reverse (confidence_ellipsoid_to_xyz), so I haven't written more tests yet.

Reply:
Yea I should have read the first post 🙈 Sorry about that!